Sources/QRCode.swift

```diff
@@ -45,10 +45,11 @@ open class QRCode {
                  encoding: String.Encoding = .utf8,
                  errorCorrectLevel: QRErrorCorrectLevel = .H,
                  withBorder hasBorder: Bool = true) {
-        guard let typeNumber = try? QRCodeType.typeNumber(of: text, errorCorrectLevel: errorCorrectLevel)
-            , let model = QRCodeModel(text: text, encoding: encoding, typeNumber: typeNumber, errorCorrectLevel: errorCorrectLevel)
+        
+        guard let model = QRCodeModel(text: text, encoding: encoding, errorCorrectLevel: errorCorrectLevel)
             else { return nil }
-        self.typeNumber = typeNumber
+//        typeNumber = try? QRCodeType.typeNumber(of: text, errorCorrectLevel: errorCorrectLevel, encoding: encoding)
+        self.typeNumber = model.typeNumber
```

Comment:
Since type number is private, and we aren't really using type number anywhere in this class, I think we can remove this property?

Reply:
Oh, sure thing, I just forget it actually